### DISCUSSION TITLE
V4.6.3

### DISCUSSION THREAD

**Discussion by WatsonYao** [WatsonYao/nian_app#25] [version]
首页右屏进展列表顶部新增，最近三天更新概览。

优化部分页面细节，修复部分反馈过来的问题，提升性能和稳定性。
修复「记本颜色」列表不显示网址本的异常。
标准记本右上角快速粘贴改为长按右下角按钮。
计时本手动录入时，底部显示最近一条历史条目，方便查看和点击使用。
评分分布，柱形图长按可以选择当前评分。
删除记账本顶部图表和按钮，列表展示所有年份记录，账目点击可选择「复制备注」。
调整部分底部弹出页的删除按钮位置和大小。
修改清单小组件的部分默认文案。
倒数日默认添加至列表顶部。
搜索和日历中出现的清单项，点击可以跳转记本并定位显示。
修改网址本标题最大显示行数到5行。
修复日历中，补录进展的顺序异常问题。
修复记账本编辑时，颜色显示异常的问题。

<hr /><em>This discussion was created from the release <a href='https://github.com/WatsonYao/nian_app/releases/tag/V4.6.3'>V4.6.3</a>.</em>

**Comment by rsuu** [comment]
请问可以给清单提供一个统计页面吗
就像计时器一样

目前计时器一次只能记录一件事情
但现实中很多事情都是可以同时做的
所以我想用清单来充当计时器

**Reply by WatsonYao** [reply]
可以展开讲讲嘛 举一些例子更好的帮助我理解使用场景

**Reply by rsuu** [reply]
比如这种半长不断的事情
```text
用清单
写一个功能: 
  日期 01-01  01-02
学习xxx: 
  日期 01-01  01-03




用计时器
写一个功能: 一小时
  休息
学习xxx: 半小时
  休息
写一个功能: 半小时
  休息
学习xxx: 半小时
写一个功能: 半小时
```

前者虽然不精确  但很方便
后者则相当繁琐

清单里的任务大多是一至两天就可以搞定的  而且是一次性的
所以没必要为了获取精确的时间而去使用计时器
而且清单的时间戳本身也足够精确  足以充当计时器

而用进度集就完全是杀鸡用牛刀了

**Reply by WatsonYao** [reply]
感觉计时本就是这么用的，清单只是个待办列表，暂无添加计时或标签等复杂化的计划。

**Reply by rsuu** [reply]
不不不，我指的是给清单添加统计页面。

比如统计每天完成了多少个任务，展示出完成了的任务，这样就方便自己调整任务调度的方式。
现在我都是数着来的(虽然不多但也有些蛋疼)。

计时器还有个问题就会被系统杀掉，这样重启后就必需重新计时，但如果不记得开始时间就很烦人了。
我是希望他的操作方式可以和清单一样，开始就是开始计时和创建事项，结束则只是关闭事项，这样就可以规避上面的问题了。

**Reply by WatsonYao** [reply]
清单本，有计划做统计页的。你这边是关心每日/每周/每月创建和完成的数量，还有其他统计指标想关注嘛~

计时本，一般都是后台运行权限的问题被杀了，取消应用的智能省电，给应用加后台锁，允许后台运行，基本就没问题，会一直记录时间。

**Reply by rsuu** [reply]
除此之外也就统计任务所花费的时间了，其他的好像也没什么了。

**Reply by WatsonYao** [reply]
目前 长按完成项是会列出创建到完成的时间间隔的 未来会显示到统计里面
